src/develop/imageop.c

```diff
@@ -458,7 +461,10 @@ int dt_iop_load_module_by_so(dt_iop_module_t *module, dt_iop_module_so_t *so, dt
   module->header = NULL;
   module->off = NULL;
   module->hide_enable_button = 0;
-  module->has_trouble = FALSE;
+  module->trouble_message = NULL;
+  module->trouble_tooltip = NULL;
+  module->trouble_message_updated = FALSE;
+  dt_pthread_mutex_init(&module->gui_lock, NULL);
```

Comment:
if explicitly initialized here, you don't need to initialize into dt_iop_default_init() above.

Reply:
Thanks! Fixed now.